src/navigation.jl

```diff
@@ -2,7 +2,7 @@
 
 module Navigation
 
-using Tk, Compat
+using Tk
```

Comment:
Yeah, there is. The problem is that I hacked together something in Gtk for private use that's not good enough to release on its own but that works too well to motivate me to work on porting ImageView. So I never actually use ImageView myself.